Cubical/Homotopy/EilenbergMacLane/CupProduct.agda

```diff
@@ -198,7 +223,7 @@ module _ {G'' : CommRing ℓ} where
   → EM (Ring→AbGroup (CommRing→Ring R)) (n +' m)
 ⌣[,,]Cₖ-syntax n m R x y = x ⌣ₖ y
 
-syntax ⌣[]ₖ-syntax R x y = x ⌣[ R ]ₖ y
-syntax ⌣[]Cₖ-syntax R x y = x ⌣[ R ]Cₖ y
+syntax ⌣[]ₖ-syntax R x y = x ⌣[ R R]ₖ y
+syntax ⌣[]Cₖ-syntax R x y = x ⌣[ R R]Cₖ y
```

Comment:
What does the extra R indicate? That it's for rings?

Reply:
Yeah. It's to avoid a clash with the syntax introduced on lines 228 and 229.